bndtools.builder/src/org/bndtools/builder/handlers/baseline/BundleVersionErrorHandler.java

```diff
@@ -60,6 +70,36 @@
                     loc = findBundleVersionHeader(bndFile);
                 }
 
+                if (loc == null) {
+                    // Not found in bnd.bnd, try build.bnd. Marker will appear on bnd.bnd
+                    IFile buildFile = Central.getWorkspaceBuildFile();
+                    loc = findBundleVersionHeader(buildFile);
+                    if (loc != null) {
+                        loc = DUMMY_LINE_LOCATION;
+                    }
+                }
+
+                if (loc == null) {
+                    // Not found in build.bnd, try ext folder. Marker will appear on bnd.bnd
+                    File extDir = new File(Central.getWorkspace().getBuildDir(), Workspace.EXT);
```

Comment:
You don't need to search the ext dir and it is also not complete. Just look at the files in `getWorkspace().getIncluded()`.